### DISCUSSION TITLE
Multi-language fallback mechanism

### DISCUSSION THREAD

**Discussion by lavnishhh** [python-pillow/Pillow#8328] [Q&A] [answered]
How do I create a fallback mechanism for different languages? I have a thing where I need to be able to display multiple fonts. If possible I'd also like to know if there is just a single font I could use for this.

**Comment by radarhere** [comment, marked as answer]
It sounds like you might be interested in #6926

**Comment by radarhere** [comment]
> If possible I'd also like to know if there is just a single font I could use for this.

Could you specify exactly what languages you're after?

**Reply by lavnishhh** [reply]
Hi! Just looked into the PR. Did not use it yet but I got the build. I was creating a generator on my server, where I came across the issue of not being able to render texts. These texts are basically originating from message data, which means it could be anything from hindi, russian, mandarin, etc.
